Mathlib/Algebra/Quaternion.lean

```diff
@@ -494,111 +511,108 @@ instance [CommSemiring S] [Algebra S R] : Algebra S ℍ[R,c₁,c₂] where
   smul_def' s x := by ext <;> simp [Algebra.smul_def]
   commutes' s x := by ext <;> simp [Algebra.commutes]
 
-theorem algebraMap_eq (r : R) : algebraMap R ℍ[R,c₁,c₂] r = ⟨r, 0, 0, 0⟩ :=
+theorem algebraMap_eq (r : R) : algebraMap R ℍ[R,c₁,c₂,c₃] r = ⟨r, 0, 0, 0⟩ :=
   rfl
 
-theorem algebraMap_injective : (algebraMap R ℍ[R,c₁,c₂] : _ → _).Injective :=
+theorem algebraMap_injective : (algebraMap R ℍ[R,c₁,c₂,c₃] : _ → _).Injective :=
   fun _ _ ↦ by simp [algebraMap_eq]
 
-instance [NoZeroDivisors R] : NoZeroSMulDivisors R ℍ[R,c₁,c₂] := ⟨by
+instance [NoZeroDivisors R] : NoZeroSMulDivisors R ℍ[R,c₁,c₂,c₃] := ⟨by
   rintro t ⟨a, b, c, d⟩ h
   rw [or_iff_not_imp_left]
   intro ht
   simpa [QuaternionAlgebra.ext_iff, ht] using h⟩
 
 section
 
-variable (c₁ c₂)
+variable (c₁ c₂ c₃)
 
 /-- `QuaternionAlgebra.re` as a `LinearMap`-/
 @[simps]
-def reₗ : ℍ[R,c₁,c₂] →ₗ[R] R where
+def reₗ : ℍ[R,c₁,c₂,c₃] →ₗ[R] R where
   toFun := re
   map_add' _ _ := rfl
   map_smul' _ _ := rfl
 
 /-- `QuaternionAlgebra.imI` as a `LinearMap`-/
 @[simps]
-def imIₗ : ℍ[R,c₁,c₂] →ₗ[R] R where
+def imIₗ : ℍ[R,c₁,c₂,c₃] →ₗ[R] R where
   toFun := imI
   map_add' _ _ := rfl
   map_smul' _ _ := rfl
 
 /-- `QuaternionAlgebra.imJ` as a `LinearMap`-/
 @[simps]
-def imJₗ : ℍ[R,c₁,c₂] →ₗ[R] R where
+def imJₗ : ℍ[R,c₁,c₂,c₃] →ₗ[R] R where
   toFun := imJ
   map_add' _ _ := rfl
   map_smul' _ _ := rfl
 
 /-- `QuaternionAlgebra.imK` as a `LinearMap`-/
 @[simps]
-def imKₗ : ℍ[R,c₁,c₂] →ₗ[R] R where
+def imKₗ : ℍ[R,c₁,c₂,c₃] →ₗ[R] R where
   toFun := imK
   map_add' _ _ := rfl
   map_smul' _ _ := rfl
 
 /-- `QuaternionAlgebra.equivTuple` as a linear equivalence. -/
-def linearEquivTuple : ℍ[R,c₁,c₂] ≃ₗ[R] Fin 4 → R :=
+def linearEquivTuple : ℍ[R,c₁,c₂,c₃] ≃ₗ[R] Fin 4 → R :=
   LinearEquiv.symm -- proofs are not `rfl` in the forward direction
-    { (equivTuple c₁ c₂).symm with
-      toFun := (equivTuple c₁ c₂).symm
-      invFun := equivTuple c₁ c₂
+    { (equivTuple c₁ c₂ _).symm with
```

Comment:
Same thing here, this just looks weird with two explicit and one implicit variable